### ISSUE TITLE
Volume Creation Wizard complains about administrative privileges

### ISSUE THREAD

**Issue by andry81** [veracrypt/VeraCrypt#1150]
![veracrypt wizard](https://github.com/veracrypt/VeraCrypt/assets/16258197/63c4ab42-1e66-4a7f-8e4d-e1365aab7b13)

But I can not run VeraCrypt as not Administrator because it always requests Administrator privileges. The `Run as Administrator` option over the executable is disabled.

# Expected behavior
Run VeraCrypt as user.

# Observed behavior
Always run as Administrator.

# Steps to reproduce
1. Run `VeraCrypt.exe`

# Your Environment

**VeraCrypt version:** 1.25-Update9

**Operating system and version:** Windows 7 x64 SP1

**System type:** 64-bit

**Comment by idrassi** [comment]
@andry81  You need to install VeraCrypt in order to avoid this beavior. Using VeraCrypt in portable mode always requires Administrator privileges in order to be able to load VeraCrypt driver dynamically since it is not installed.

**Comment by idrassi** [comment]
@andry81 Now I noticed the fact that "Run as Administrator" menu in Windows is disabled for your user account. Based on this, it seems that your Windows 7 system might be running with User Account Control (UAC) disabled. This results in all executables being run with full administrative privileges by default, which poses security risks.

To confirm this, can you please check UAC Settings:

- Open the Control Panel.
- Navigate to "User Accounts and Family Safety" > "User Accounts".
- Click on "Change User Account Control settings".
- If the slider is set to "Never notify", this means UAC is disabled. It's recommended to set the slider to, at minimum, the default level ("Notify me only when programs try to make changes to my computer") or even a level higher if you desire extra caution.

Does this solve the issue?

**Comment by andry81** [comment]
> If the slider is set to "Never notify", this means UAC is disabled. It's recommended to set the slider to, at minimum, the default level ("Notify me only when programs try to make changes to my computer") or even a level higher if you desire extra caution.

> Now I noticed the fact that "Run as Administrator" menu in Windows is disabled for your user account.
No, it is not enabled for the executable in the Windows Explorer options dialog. But the Windows does request for the Administrator privileges even if not enabled. This is why the Creation Wizard complains.
